### PR TITLE
main: refactor to only catch exceptions for parseConfigFile

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

**Review by TrustHenry — APPROVED** [review]
LGTM

File: source/agora/node/main.d [hunk 1]
```diff
@@ -63,25 +63,30 @@ private int main (string[] args)
...
 
-        runTask(() => node_listener_tuple = runNode(config));
-    }
-    catch (Exception ex)
+    if (cmdln.config_check)
     {
-        writefln("Failed to parse config file '%s'. Error: %s",
-            cmdln.config_path, ex.message);
-        return 1;
+        writefln("Config file '%s' succesfully parsed.", cmdln.config_path);
```
Comment (AndrejMitrovic) [inline source/agora/node/main.d]:
```suggestion
        writefln("Config file '%s' successfully parsed.", cmdln.config_path);
```

Why do we want a chatty node?

File: source/agora/node/main.d [hunk 2]
```diff
@@ -63,25 +63,30 @@ private int main (string[] args)
             ex.message);
     }
 
-    NodeListenerTuple node_listener_tuple;
-    try
-    {
-        auto config = parseConfigFile(cmdln);
-        if (cmdln.config_check)
+    bool configError;
```
Comment (AndrejMitrovic) [inline source/agora/node/main.d]: Not according to our style guide..

Comment (Geod24) [reply source/agora/node/main.d]: Oops

**Comment by Geod24** [comment]
Fixed

**Review by AndrejMitrovic — APPROVED** [review]
This code was changed so many times that I don't even have an opinion on it anymore.

**Pull request merged**